charts/platform-service/templates/deployment.yaml

```diff
@@ -176,6 +187,12 @@ spec:
 {{ .Values.health.startupProbe | toYaml | trim | indent 12 }}
 {{- end }}
           volumeMounts:
+{{- if eq .Values.keyVaultSecrets.enabled true }}

```

Comment:
Similar comment as to the env keyvault section... once they are k8s secrets we can use regular syntax for mounting them as volumes. So no need for this section I think anymore.